api/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7
+
+LABEL Author="Brylie Christopher Oxley"
+LABEL E-mail="brylie.oxley@maas.global"
+LABEL version="0.0.1"
+
+LABEL Name=sustainable-mobility-api Version=0.0.1
+EXPOSE 8080
+
+WORKDIR /app
+ADD . /app
```

Comment:
Use [`COPY` instead of `ADD`](https://www.ctl.io/developers/blog/post/dockerfile-add-vs-copy/)